packages/fast-usdc/src/util/bank.js

```diff
@@ -0,0 +1,14 @@
+/* global globalThis */
+
+export const queryUSDCBalance = async (
+  address,
+  api,
+  denom,
+  fetch = globalThis.fetch,
```

Comment:
this is provided in every instance. please require it

Reply:
Done